source/client-side-encryption/tests/bulk.template.yml

```diff
@@ -0,0 +1,63 @@
+runOn:
+  - minServerVersion: "4.1.10"
+database_name: &database_name "default"
+collection_name: &collection_name "default"
+
+data: []
+json_schema: {{schema()}}
+key_vault_data: [{{key()}}]
+
+tests:
+  - description: "Bulk write with encryption"
+    clientOptions:
+      auto_encrypt_opts:
+        kms_providers:
+          aws: {} # Credentials filled in from environment.
+    operations:
+      - name: bulkWrite
+        arguments:
+          requests:
+            - name: insertOne
```

Comment:
Consider testing the other request types as well.

Reply:
Added updateOne and deleteOne